x-pack/test/functional/apps/maps/import_geojson/add_layer_import_panel.js

```diff
@@ -7,18 +7,24 @@
 import expect from '@kbn/expect';
 import path from 'path';
 
-export default function ({ getPageObjects }) {
+export default function ({ getPageObjects, getService }) {
   const PageObjects = getPageObjects(['maps', 'common']);
 
   const IMPORT_FILE_PREVIEW_NAME = 'Import File';
   const FILE_LOAD_DIR = 'test_upload_files';
   const DEFAULT_LOAD_FILE_NAME = 'point.json';
+  const security = getService('security');
 
   describe('GeoJSON import layer panel', () => {
     before(async () => {
+      await security.testUser.setRoles(['global_maps_all', 'geoall_data_writer']);
```

Comment:
My one concern here is that we've seen problems where when we switch roles and the default behavior is to refresh the page, depending on the previous test you could be on a page where this new roles cause an error.  We've gone back and switched several of those cases to pass the `false` parameter to not refresh the page since the next navigation step would refresh it anyways. 
But if it passes CI I'm OK with leaving it as-is.

Reply:
Thanks @LeeDr I did the adjustment for the other testfile in this folder. Thanks!